src/ert/cli/main.py

```diff
@@ -37,7 +37,7 @@ def run_cli(args):
     logger = logging.getLogger(__name__)
     logger.info(
         "Logging forward model jobs",
-        extra={"workflow_jobs": str(res_config.forward_model.job_name_list())},
+        extra={"workflow_jobs": str(res_config.forward_model_job_name_list())},
```

Comment:
Maybe just `extra={"workflow_jobs": str([job.name for job in res_config.forward_model_list]}`.